### PR TITLE
Add :ssl smtp setting to ActionMailer::Base rdoc

### PR DESCRIPTION
**Pull request by pedroadame**
### Summary

Added `:ssl` documentation in `smtp_settings` of `ActionMailer::Base`. It works but it doesn't appeared before in the rdoc.

I'm creating this PR because I needed the use of `:ssl` and I had to dig into the web just to find that it exists. If not set, mailing with NameCheap's privateemail doesn't work even with `enable_starttls_auto` set to `true` 

I think it would be useful if people could read about it in the documentation.

### CODE REVIEW

File: actionmailer/lib/action_mailer/base.rb [hunk 1]
```diff
@@ -392,6 +392,7 @@ module ActionMailer
   #     of an OpenSSL verify constant (<tt>'none'</tt>, <tt>'peer'</tt>, <tt>'client_once'</tt>,
   #     <tt>'fail_if_no_peer_cert'</tt>) or directly the constant (<tt>OpenSSL::SSL::VERIFY_NONE</tt>,
   #     <tt>OpenSSL::SSL::VERIFY_PEER</tt>, ...).
+  #     <tt>:ssl</tt> - Forces ActionMailer to use SSL in the SMTP connection. Defaults to <tt>false</tt>.
```
Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: We are also missing `:tls` for this option. Lets change to:

`<tt>:ssl/:tls</tt> Enables Action Mailer to use SMTP/TLS (SMTPS: SMTP over direct TLS connection)`

Note, there is no default.

Comment (pedroadame) [inline actionmailer/lib/action_mailer/base.rb]: Done!

**Comment by vipulnsward** [comment]
Thanks, can you squash your commits?
Also add a `[ci skip]` to the commit message, so we avoid running tests on Travis.

**Comment by pedroadame** [comment]
Opened a new PR due to unability of squash.

**Pull request closed without merging**